src/client/entity-editor/alias-editor/alias-editor.js

```diff
@@ -37,15 +35,14 @@ import {faQuestionCircle} from '@fortawesome/free-solid-svg-icons';
  * @param {Array} props.languageOptions - The list of possible languages for an
  *        alias.
  * @param {Function} props.onClose - A function to be called when the button to
- *        close the editor is clicked.
+ *        add new alias is clicked.
  * @param {boolean} props.show - Whether or not the editor modal should be
  *        visible.
  * @returns {ReactElement} React element containing the rendered AliasEditor.
  */
 const AliasEditor = ({
 	languageOptions,
-	onClose,
-	show
+	onClose
```

Comment:
The name onClose needs to be changed everywhere since the function has changed.
What does it do now?

Reply:
ok so this onClose is a dispatch function which is used to remove any empty alias row when user click on Done button (refer SS), 
I thought to change the name to onDone but it doesn't seem very descriptive to me so I thought it may be good to hear some review